jetty-deploy/src/main/java/org/eclipse/jetty/deploy/jmx/DeploymentManagerMBean.java

```diff
@@ -95,9 +95,10 @@ private String toRef(App app)
         return apps;
     }
 
-    public Collection<AppProvider> getAppProviders()
+    @ManagedAttribute("Registered AppProviders")
+    public List<String> getAppProviders()
     {
-        return _manager.getAppProviders();
+        return _manager.getAppProviders().stream().map((ap) -> ap.toString()).collect(Collectors.toList());
```

Comment:
Just a nit, you can use `.map(String::valueOf)` here.

Reply:
done